### DISCUSSION TITLE
Turn off the hotend fan if paused for too long

### DISCUSSION THREAD

**Discussion by rraallvv** [mriscoc/Ender3V2S1#1342] [Ideas]
When a print job takes too long, I sometimes have to pause the printer overnight. However, the hotend fan keeps running even though the hotend has cooled down to room temperature. It would be great if the firmware had an option to automatically turn off the fan when the hotend reaches a safe temperature during an indefinite pause.

**Comment by mriscoc** [comment]
For 4.2.2 and 4.2.7 printer boards only the part cooling fan is firmware controllable, while the hotend and electronics fans are not.

**Reply by rraallvv** [reply]
Thanks for the quick reply.
